internal/cosesign1/Makefile

```diff
@@ -0,0 +1,94 @@
+#
+# This makefile lets us exercise the library via the sign1util tool and shows various examples of
+# creating and checking COSE Sign1 documents as well as getting DID:x509 documents out of them
+# Note that today the PEM files that match are not checked in as the github/ADO credential checker will trigger.
```

Comment:
This documentation (and general usage information) might be better in a README instead of requiring a user to look through the makefiles to figure it out.

Reply:
That is the README with the sign1util. This is telling people what this actual Makefile does. The alternative would be to have  these command lines in a document and then the user have to copy and paste them out and run them by hand or make their own Makefile. Since we already had a makefile to do the dev work committing it seems like the best idea and given it is being committed adding a bit of information about what it is doing seems to be genuinely helpful.